libs/blocks/gnav/gnav.js

```diff
@@ -174,29 +185,64 @@ class Gnav {
     linkGroups.forEach((linkGroup) => {
       const image = linkGroup.querySelector('picture');
       const anchor = linkGroup.querySelector('p a');
-      const title = anchor.textContent;
+      const title = anchor?.textContent;
       const subtitle = linkGroup.querySelector('p:last-of-type');
       const titleWrapper = createTag('div');
+      titleWrapper.className = 'link-group-title';
       anchor.href = makeRelative(anchor.href, true);
       const link = createTag('a', { class: 'link-block', href: anchor.href });
 
       linkGroup.replaceChildren();
       titleWrapper.append(title, subtitle);
-      const contents = !image ? [titleWrapper] : [image, titleWrapper];
+      const contents = image ? [image, titleWrapper] : [titleWrapper];
       link.append(...contents);
       linkGroup.appendChild(link);
     });
   };
 
+  setMenuAnalytics = (el) => {
+    switch (el.nodeName) {
+      case 'DIV':
+        if (el.classList.contains('link-group')) {
+          const title = el.querySelector('.link-group-title')?.innerHTML.split('<')?.[0].trim();
```

Comment:
This is pretty brittle, but I couldn't figure out any other way to get just the title from the link.  Using .textContent returns the text with no separator between the title and text. 

This gets the html and grabs the text before the `<p>` tag.

@seanchoi-dev if you have any suggestions I'd like something more robust.